steal.lic

```diff
@@ -408,20 +372,20 @@ class Steal
       update_target(target, 'exceptionally') if count == 1
       bin_item(target['item'])
     when /but decide you have too many outstanding fines as is/
-      beep
+      DRC.beep
       echo('Too many fines, cannot steal anything')
```

Comment:
Anything worth beeping should probably use DRC.message for more visibility (since I know for me at least, beep doesn't work).

Reply:
Agree, beep doesn’t work for me either

Reply:
I replaced the beeps with DRC.message